content/docs/user-guide/gems/reference/ros2/_index.md

```diff
@@ -0,0 +1,119 @@
+
+Use this helpful command to install:
+
+```
+sudo apt install ros-${ROS_DISTRO}-ackermann-msgs ros-${ROS_DISTRO}-control-toolbox ros-${ROS_DISTRO}-nav-msgs ros-${ROS_DISTRO}-gazebo-msgs
+```
+
+## Features
+
+* Direct and natural support of ROS2 ecosystem:
+  * No bridges. Your simulation node will function as any other ROS2 node.
+    * This is also good for performance
+  * Easy way to include ROS2 dependencies.
+* Sensors:
+  * Sensor Component serving as a handy abstraction.
+  * Example implementations of Lidar, Camera, IMU sensors.
+    * Including a few Assets and prefabs which are ready to use. 
+* Automated handling of:
+  * Simulation time, publishing `/clock` supporting non-real time.
+  * Publishing of transformation frames (`/tf`, `/tf_static`).
+  * Validation for topic and namespace names.
+* Robot Control Component:
+  * A quick to use method of controlling your robot with Twist messages.
+  * Can be used with custom LUA scripting. 
+
+For a "feel" of these features, see an [example project](https://github.com/o3de/RobotVacuumSample) which uses this Gem to run navigation stack.
+
+<!-- ## Clone this repository
+
+Clone the gem repository from Github:
+
+```
+git clone https://github.com/RobotecAI/o3de-ros2-gem.git
+```
+
+## Building
+
+The Gem is built through building a project which uses it. Make sure to
+[source your ros2 workspace](https://docs.ros.org/en/rolling/Tutorials/Configuring-ROS2-Environment.html#source-the-setup-files)
+before building.
+
+## Adding Gem to your project
+
+To use this Gem in your project, you need to register the Gem with O3DE. First, clone this repository. Then,
+in o3de folder:
+```
+scripts/o3de.sh register --gem-path <PATH_TO_CLONED_ROS2_GEM>
+scripts/o3de.sh enable-gem -gn ROS2 -pp <PATH_TO_YOUR_PROJECT>
+```
+
+## Example project
+
+You can test O3DE ROS2 Gem with [this project](https://github.com/RobotecAI/Ros2WarehouseDemo).
+It will allow you to run robot navigation. All necessary assets are included. -->
+
+## User Guides
+
+Follow the [ROS 2 Gem User Guide](ros2-gem-user-guide.md) to understand its concepts and components.
+
+<!-- If you plan on contributing please follow the [Pull Request Publishing](docs/guides/pr_publishing.md) guide. For those using the Clion IDE we advise to follow the [Development in Clion](docs/guides/development_in_clion.md) guide. -->
+
+## How to create your own robotic simulation
+
+>This section is to be detailed.
+
+Once you are set up and familiar with the example project, consider the following steps:
+1. [Create a new O3DE project](/docs/welcome-guide/create/) 
+2. Register ROS2 Gem and other useful Gems to created project. 
+Follow [Registering Gems to a Project](/docs/user-guide/project-config/register-gems/) guide. 
+3. Create or import Assets for your robots and environment. 
+   1. You can use formats supported by O3DE.
+   2. You can import your robot from URDF.
+   3. Imported models might require some adjustments to be simulation-ready.
```

Comment:
Add a point about vehicle dynamics - mobilizing your robot and selecting / configuring control method and dynamics parameters.